magefile.go

```diff
@@ -72,3 +85,58 @@ func Notice() error {
 		filepath.Join("dev-tools", "templates", "notice", "NOTICE.txt.tmpl"),
 	)
 }
+
+// GoImports executes goimports against all .go files in and below the CWD.
+func GoImports() error {
```

Comment:
Should we move this to `elastic-agent-libs`?

Reply:
Sure, we can add a Format target there if there isn't one already that includes this.